README.md

```diff
@@ -58,11 +58,11 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [Leaderboard: setup project] <a name="about-project"></a>
+# 📖 [Leaderboard: final touches] <a name="about-project"></a>
 
 > Describe your project in 1 or 2 sentences.
 
-**[Leaderboard: setup project]** is a solo project where scores of different people from a competition can be recorded
+**[Leaderboard: final touches]** is a solo project where scores of different people from a competition can be recorded
 
 ## 🛠 Built With <a name="VS Code"></a>
 
```

Comment:
- [ ] Please put the steps and commands required to install and run your application on the README file. ✍️

<img src="https://user-images.githubusercontent.com/17243325/216654501-dcc6bc92-cc08-445a-9b06-2d6ee6d3c2af.png" width="400">